infrastructure/kyverno/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - https://raw.githubusercontent.com/kyverno/kyverno/v1.5.5/definitions/release/install.yaml
+  - https://raw.githubusercontent.com/kyverno/kyverno/release-1.6/config/install.yaml
```

Comment:
Please undo this change, Kyverno releases are tested in CI and committed back to this repo with GH Actions.

Reply:
We need version `1.6` for the cosign roots verification, otherwise we get:
```
ClusterPolicy/verify-flux-images dry-run failed, error: failed to create typed patch object: errors:                            False    
                                                          .spec.rules[0].verifyImages[0].roots: field not declared in schema                                                                     
                                                          .spec.rules[0].verifyImages[1].roots: field not declared in schema                                                                     
                                                          .spec.rules[0].verifyImages[2].roots: field not declared in schema                                                                     
                                                          .spec.rules[0].verifyImages[3].roots: field not declared in schema                                                                     
                                                          .spec.rules[0].verifyImages[4].roots: field not declared in schema                                                                     
                                                          .spec.rules[0].verifyImages[5].roots: field not declared in schema 
```

Reply:
Ok so set a fix version that CI will keep up to date, like so:
```
https://raw.githubusercontent.com/kyverno/kyverno/v1.6.0-rc2/definitions/release/install.yaml
```

Reply:
The new version seems to have moved the `install.yaml` to the `config` dir. But just updated it to lean on tags instead as advised. PTAL